### ISSUE TITLE
XAML changes on iOS simulator not detected when running the sample code from VS 

### ISSUE THREAD

**Issue by supreettare** [UXDivers/Gorilla-Player-Support-And-Issues#44]
Here are the details: 
Using VS 2015 
windows: Gorilla Player 0.8.2.2, server 0.8.2.2 
mac: Gorilla Player 0.8.2.2, server 0.8.2.2 

Player instance installed in iOS simulator running on MAC (iPhone 5s, iOS 9.1)

Gorila player connected to the server on windows machine, auto detect did not work, connected with IP & Port. Both Mac & Windows are on same network. So I am past the connectivity issues. 

I fire open SampleData app that ships with the player i=on windows using VS 2015 

open JsonDataPage.xaml (I notice the gorilla icon on top right, so this is working so far) 

make some small changes to the XAML (change the width of the column to 200. Vertical Alighnment to Start etc.) 

No changes reflected in the app running on iOS simulator. 

Is this a known issue? Am I missing something? What version of Xamarin should this work on? 
Xamarin Forms version 2.0.0.6484 installed on portable, android & iOS

**Comment by supreettare** [comment]
you need to look for the XAML changes in Gorilla Player & not the actual app itself, I should have thought of that before :) 
I guess it will be great if you add this as a foot note in your getting started documentation.

**Comment by jpbrocca** [comment]
Hi @supreettare, thanks for your suggestion.

You were expecting to see the preview on the "the actual app itself"...what you mean by that?
(I am trying to find the root cause of what could have confused that to improve our communication and avoid this to happen to other users).
Thanks for helping us to make a better product!

Best,
JP

**Comment by supreettare** [comment]
Hwy @jpbrocca 
Yes, I was expecting to the changes in my actual app running on simulator. So instead of firing gorilla app on the simulator, I kept open my app & was waiting for it to reflect the changes. 
It was by accident when I closed my app & the simulator presented to me the Gorilla Player where I noticed the changes I was making, I got to know that you must look for the changes in Gorilla Player. 

Will be great if you put a highlighted note saying "Your changes will appear on Gorilla Player app running on the simulator or device"  or something like that. 

That aside, your sample apps worked, now when I open my app & I was getting an issue. I will log a new bug for that. 

Thanks 
ST
